ycsb-rs/src/main.rs

```diff
@@ -106,22 +127,66 @@ fn main() -> Result<()> {
             for _client_idx in 0..actual_client_count {
                 let database = database.clone();
                 let db = db::create_ob(&database, config.clone()).unwrap();
+                // count the ops per client
+                let counter = Arc::new(AtomicUsize::new(0));
+                db_counters.push(counter.clone());
                 for _ in 0..props.obkv_client_reuse {
                     let db = db.clone();
                     let wl = wl.clone();
                     let cmd = cmd.clone();
                     let runtime = runtimes.default_runtime.clone();
+                    let counter_clone = Arc::clone(&counter);
```

Comment:
`Arc::clone` and `.clone` are all ok.
Seem `.clone` is used above, maybe follow the same style here?

Reply:
Nice suggestion, fixed